.github/workflows/pr.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
I think PRs don't have to be limited to a branch

Reply:
oh right.  This isn't the target branch?